make/run-cicd-script.mk

```diff
@@ -1,5 +1,5 @@
 
-OWNER_AND_BRANCH_LOCATION=codeready-toolchain/toolchain-cicd/master
+OWNER_AND_BRANCH_LOCATION=mfrancisc/toolchain-cicd/feature/ASC-245_cluster_labels
```

Comment:
reminder :)

Reply:
changed in [2c13a2f](https://github.com/codeready-toolchain/toolchain-e2e/pull/645/commits/2c13a2fa53fac325af469c3435cf09662f097ce4)